js/libs/builder.js

```diff
@@ -353,17 +515,21 @@ var app = new Vue({
           Fliplet.Studio.emit('widget-save-label-reset');
           Fliplet.Studio.emit('widget-info-label-update');
           setTimeout(() => {
-            $vm.setupCodeEditor()
-          }, 1)
+            $vm.setupCodeEditor();
+          }, 1);
         }
 
         return;
       }
 
       // Save and close
-      $vm.save().then(function() {
+      $vm.save().then(() => {
         Fliplet.Widget.complete();
       });
     }
+
+    Fliplet.API.request('v1/user').then(function(response) {
+      window.userData = response.user;
```

Comment:
@squallstar I need to use that data later in the code, how can I set the var on a global level to the entire `builder.js` code but keep it private?